### DISCUSSION TITLE
Why does tsconfig use esnext？

### DISCUSSION THREAD

**Discussion by winixt** [vuejs/rfcs#332] [General]
Why is the tsconfig compilerOptions target set to `esnext` instead of `es2015` and won't the compiled code have huge compatibility issues if the latest es syntax is used in the source code?

**Comment by IndexXuan** [comment]
because after tsc, we have babel to compile again and add polyfill.

**Reply by winixt** [reply]
Where can I find this？I run `build`, it can't compile `async`.

**Reply by IndexXuan** [reply]
I think you can config the browserlist.

**Comment by leopiccionia** [comment]
It seems to have been changed to ES2016. See the following commits:
* https://github.com/vuejs/vue-next/commit/2788154f7707928f1dd3e4d9bd144f758a8c0478
* https://github.com/vuejs/vue-next/commit/438754a0d1428d10e27d1a290beb4b81da5fdaeb
